functionnal_tests/cypress/integration/app_workspace_advanced/operation_workspace_advanced.js

```diff
@@ -0,0 +1,39 @@
+import { SELECTORS as s } from '../../support/generic_selector_commands'
+import { PAGES as p } from '../../support/urls_commands'
+
+describe('app workspace advanced', function () {
+    const newDescription = 'myNewDescription'
+    const workspaceId = 1
+
+    before(() => {
+        cy.resetDB()
+        cy.setupBaseDB()
+    })
+
+    beforeEach(function () {
+        cy.loginAs('administrators')
+        cy.visitPage({pageName: p.DASHBOARD, params: { workspaceId }})
+    })
+
+    it('Change workspace description', function () {
```

Comment:
the it() should be more detailed about the test it checks.
It should be something like:
``` javascript
describe('App Workspace Advanced', () => {
  describe("Changing the workspace's description"), () => {
    it('Should update the description in the dashboard', () => {
      // the test
    })
  })
})
```